docs/4.3/installation.md

```diff
@@ -37,45 +9,77 @@ The following examples install the 64-bit version of Teleport binaries, but
+    $ cd teleport
+    $ ./install
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
+
+=== "DEB"
+
+    ```bash
+    $ curl https://get.gravitational.com/teleport_{{ teleport.version }}_amd64.deb.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport_{{ teleport.version }}_amd64.deb
+    $ sha256sum teleport_{{ teleport.version }}_amd64.deb
+    # Verify that the checksums match
+    $ dpkg -i teleport_{{ teleport.version }}_amd64.deb
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
+
+=== "RPM"
+
+    ```bash
+    $ curl https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
+    $ sha256sum teleport-{{ teleport.version }}-1.x86_64.rpm
+    # Verify that the checksums match
+    $ rpm -i teleport-{{ teleport.version }}-1.x86_64.rpm
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
+
+=== "Amazon Linux 2"
+
+    ```bash
+    $ wget https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
+    $ tar -xzf teleport-v{{ teleport.version }}-1.x86_64.rpm
+    $ cd teleport
+    $ sudo ./install
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
+
+=== "ARMv7"
+
+    ```bash
+    $ curl https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz
+    $ shasum -a 256 teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz
+    $ cd teleport
+    $ ./install
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
 
-**Tarball**
-```bash
-$ curl https://get.gravitational.com/teleport-v{{ teleport.version }}linux-amd64-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-$ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-# Verify that the checksums match
-$ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-$ cd teleport
-$ ./install
-$ which teleport
-/usr/local/bin/teleport
-```
+## Docker
 
-**RPM**
-```bash
-$ curl https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
-$ sha256sum teleport-{{ teleport.version }}-1.x86_64.rpm
-# Verify that the checksums match
-$ rpm -i teleport-{{ teleport.version }}-1.x86_64.rpm
-$ which teleport
-/usr/local/bin/teleport
-```
+Please follow our [OSS Docker Quickstart](docs/4.3/quickstart_docker.md) or [Enterprise Docker Quickstart](enterprise/quickstart-enterprise/#run-teleport-enterprise-using-docker) for install and setup instructions.
```

Comment:
```suggestion
Please follow our [OSS Docker Quickstart](quickstart_docker.md) or [Enterprise Docker Quickstart](enterprise/quickstart-enterprise/#run-teleport-enterprise-using-docker) for install and setup instructions.
```

(fix docs link, whoops, my bad)

Reply:
Can you make this `enterprise/quickstart-enterprise.md#run-teleport-enterprise-using-docker`

Reply:
Done